serverless/event_workflows/serverless-using-brokers.adoc

```diff
@@ -49,8 +49,19 @@ spec:
 
 However, the Knative CLI (`kn`) provides the following commands to simplify creating and managing triggers.
 
+.Prerequisites
+
+Before you can use triggers, you will need:
+
+* Knative Eventing and `kn` installed.
+* An available broker, either the `default` broker or one that you have created.
++
+You can create the `default` broker either by following the instructions on xref:../serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing], or by using the `--inject-broker` flag while creating a trigger. Use of this flag is described in the procedure below.
+* An available event consumer, for example, a Knative service.
```

Comment:
s/is described in the procedure below./is described later in this section./

**below** _adj_
Do not use to indicate a relative location in a document, as in “the information below.” Use “the information
later in this section.”